static/script.js

```
@@ -8,6 +8,14 @@
 
 // get read history page id and title from localStorage
 let readHistory = JSON.parse(localStorage.getItem("readHistory")) || [];
+// get Url check is it localhost
+const isLocalhost =
+    window.location.hostname === "localhost" ||
+    window.location.hostname === "127.0.0.1";
+const startAds = isLocalhost
+    ? () => (adsbygoogle = window.adsbygoogle || []).push({})
```

Biome (1.9.4)

[error] The assignment should not be in an expression.

The use of assignments in expressions is confusing.
Expressions are often considered as side-effect free.

(lint/suspicious/noAssignInExpressions)